<?xml version="1.0"?>
<launch>



<include file="$(find setup)/launch/p3at_hmapping.launch">
      <arg name="robotname" value="robot0"/>
</include>
<include file="$(find setup)/launch/p3at_move_base2.launch">
      <arg name="robotname" value="robot0"/>
</include>


<include file="$(find setup)/launch/p3at_hmapping.launch">
      <arg name="robotname" value="robot3"/>
</include>
<include file="$(find setup)/launch/p3at_move_base2.launch">
      <arg name="robotname" value="robot3"/>
</include>




</launch>
